src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String DIVIDING_LINE = "________________________________________";
```

Comment:
Constant names are all in upper case with underscore separating words. Well done!